components/combobox/stories/combobox.mdx

```diff
@@ -0,0 +1,89 @@
+import { Canvas, ArgTypes, Meta, Description, Title } from '@storybook/blocks';
+
+import * as ComboboxStories from './combobox.stories';
+
+<Meta of={ComboboxStories} title="Docs" />
+
+<Title of={ComboboxStories} />
+<Description of={ComboboxStories} />
+
+## Usage notes
+### General notes
+- Combobox uses `.spectrum-PickerButton` instead of a `.spectrum-Picker`
+- The following classes must be added:
+  - `.spectrum-Combobox-textfield` is required on the Textfield outer element (`.spectrum-Textfield`)
+  - `.spectrum-Combobox-input` is required on the `<input>` element inside of Textfields (`.spectrum-Textfield-input`)
+  - `.spectrum-Combobox-button` is required on the FieldButton (`.spectrum-ActionButton spectrum-ActionButton--sizeM`)
+
+### Component separated from InputGroup
+This component was previously a variant style for InputGroup. **InputGroup is deprecated**. The classes containing `InputGroup` have been renamed or removed:
+- `.spectrum-InputGroup-textfield` -> `.spectrum-Combobox-textfield`
+- `.spectrum-InputGroup-input` -> `.spectrum-Combobox-input`
+- `.spectrum-InputGroup-button` -> `.spectrum-Combobox-button`
+- `.InputGroup` -> removed
+
+### Indicating validity and focus
+Validity and focus must be bubbled up to the parent so descendants siblings can be styled. Implementations should add the following classes to the `.spectrum-Combobox` parent class in the following situations:
+
+- `.is-focused` - when the input or button is focused with the mouse
+- `.is-keyboardFocused` - when the input or button is focused with the keyboard
+- `.is-valid` - when the input has an explicit valid state
+- `.is-invalid` - when the input has an explicit invalid state
+- `.is-disabled` - when the control is disabled; should also add to the `.spectrum-Combobox-textfield` and include a `[disabled]` attribute to the `.spectrum-Combobox-button`
+- `.is-loading` - when the progress circle is being shown
+
+## Standard variants
+
+### Standard - open
+
+<Canvas of={ComboboxStories.Default} />
+
+### Standard - with label
+
+<Canvas of={ComboboxStories.WithLabel} />
```

Comment:
Good callout. I was able to adjust this so that the default height is 220px, which removes the need to set the height or `inline: false` on this one 🎉